pkg/config/testdata/protocol.test.yml

```diff
@@ -0,0 +1,4 @@
+ProtocolConfiguration:
+  NativeActivations:
+    - ContractManagement: [0]
```

Comment:
This differs in syntax from other config changes. Dashes make `NativeActivations` an array of single-key objects. Could you recheck this one?

Reply:
What's bad with it being an array?

Reply:
https://github.com/nspcc-dev/neo-go/pull/1827#discussion_r592429532. And array YAML structure is more complex, like
```
ProtocolConfiguration:
  NativeActivations:
    - Name: ContractManagement
      History: [0]
```
And I think nothing else.

So turn it back to array?

Reply:
Syntactically current scheme looks good to me.

Reply:
Except it seems to be changed again.